download_models/downloadOVMSModels.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#todo: update for how we get the bit model
```

Comment:
what is bit model?

Reply:
@jstein13 can you explain what's bit model?